app/src/main/java/ch/epfl/sdp/blindly/audio/AudioStorage.kt

```diff
@@ -0,0 +1,34 @@
+package ch.epfl.sdp.blindly.audio
+
+import android.net.Uri
+import android.util.Log
+import android.widget.Toast
+import com.google.firebase.storage.FirebaseStorage
+import com.google.firebase.storage.UploadTask
+import dagger.hilt.android.AndroidEntryPoint
+import java.io.File
+import javax.inject.Inject
+
+open class AudioStorage @Inject constructor(val storage: FirebaseStorage) {

```

Comment:
It's also coming in my PR #227 as `Recordings`, which one should we keep?

Reply:
Yours I think, I'll merge everything after in my pr, it'll be cleaner this way